internal/xdscache/v3/listener.go

```diff
@@ -134,15 +134,17 @@ type ListenerConfig struct {
 	TracingConfig *TracingConfig
 }
 
+type ExtensionServiceConfig struct {
+	ExtensionService client.ObjectKey
```

Comment:
we could probably change this (back) to be a NamespacedName (even though ObjectKey is an alias to it) just to limit import blast radius etc.

Reply:
(so we don't import controller-runtime packages into this one unnecessarily for example)

Reply:
Looking into it....

Reply:
Fixed...